PHP trace correlation identifiers - try to make it a little clearer (#25755)
DataDog/documentation: content/en/tracing/other_telemetry/connect_logs_and_traces/php.md

Log::debug('Hello, World!');
```

The PHP tracer adds the available trace correlation identifiers to the log context, in JSON format. The logged message above could be transformed into:

```
[2022-12-09 16:02:42] production.DEBUG: Hello, World! {"dd.trace_id":"1234567890abcdef","dd.span_id":"1234567890abcdef","dd.service":"laravel","dd.version":"8.0.0","dd.env":"production","status":"debug"}
```

**Note**: If there is a placeholder in your message or if a trace ID is already present in the message, the PHP tracer does **not** add the trace correlation identifiers to the log context.

#### Option 2: Use placeholders in your message {#use-placeholders-in-your-message}

> You can use placeholders in your message to select which trace correlation identifiers will be automatically injected into your logs. The PHP tracer supports the following placeholders:

Reviewer: ```suggestion
You can use placeholders in your message to select which trace correlation identifiers are automatically injected into your logs. The PHP tracer supports the following placeholders:
```

Log::info('Hello, World! [%dd.trace_id% %dd.span_id% %status%]');
```

> The PHP tracer replaces the placeholders with the corresponding values. For example, the logged message above will be transformed into:

Reviewer: ```suggestion
The PHP tracer replaces the placeholders with the corresponding values. For example, the logged message above could be transformed into:
```

Reviewer: Thanks for clarifying @cecile75, I've updated the recommendation. My point here is to avoid use of future tense, which is not recommended per the style guide.

Should this also be updated on line `58`, which likewise uses `could be`?

Author: line 58 is little different in my opinion.
With option 1, any trace correlation identifier with a value at the time of the log will be injected. So it's really more a "could" as the user doesn't chose explicitly which trace correlation identifiers will be injected.
While with option 2, as the coder explicitly choose to inject %dd.trace_id% %dd.span_id% %status% , I think we can assume that these items have a value at this time of the log and will be injected and present in the resulting log.
